megamek/i18n/megamek/common/options/messages.properties

```diff
@@ -140,6 +140,8 @@ GameOptionsInfo.option.allow_nukes.displayableName=Allow command-line nukes.
 GameOptionsInfo.option.allow_nukes.description=This must be checked to allow players to throw nukes from the command line. If this is not checked, nukes are still available as advanced munitions.
 GameOptionsInfo.option.really_allow_nukes.displayableName=REALLY allow command-line nukes.
 GameOptionsInfo.option.really_allow_nukes.description=This must be checked to allow players to throw nukes from the command line. If this is not checked, nukes are still available as advanced munitions.
+GameOptionsInfo.option.gm_can_kill_units.displayableName=Allow command-line kill (GM only).
```

Comment:
I suggest lumping this together with command line nukes. I doubt the distinction is required.

Reply:
Personally I would go with "GM gets access to all GM commands by default". Nominally, the point is to give them access to various tools that let them arbitrarily alter the scenario as needed. The GM status itself already has some safeguards applied to prevent abuse (can't just activate it midgame without the players voting to allow it, etc.).